packages/modules/src/modules/banners/common/BannerText.tsx

```diff
@@ -24,6 +24,34 @@ type BannerTextProps = {
     children?: React.ReactNode;
 };
 
+export const createBannerBodyCopy = (
+    paras: (Array<JSX.Element> | JSX.Element)[],
+    highlights: Array<JSX.Element> | JSX.Element | null | undefined,
+    renderStyles: BannerTextStyles,
+): JSX.Element[] | JSX.Element => {
+    const parasToProcess = Array.isArray(paras) ? paras : [paras];
+    const len = parasToProcess.length - 1;
+
+    if (len < 0) {
```

Comment:
storing `length-1` as `len` is quite confusing for humans!
Why not `if (parasToProcess.length === 0)` here?

Reply:
Renamed variables to help make functionality clearer